tests/basic/jsonpath-assertion-count.ref

```diff
@@ -0,0 +1,44 @@
+{
+  "answerElements" : [
+    {
+      "class" : "org.batfish.question.jsonpath.JsonPathQuestionPlugin$JsonPathAnswerElement",
+      "results" : {
+        "0" : {
+          "assertionResult" : false,
```

Comment:
Looking at the bugfix, it seems you just set summary false in the inner questions in the tests. so the bug was that the inner question had summary set to true?

That seems error-prone – how would we tell that the assertion count=0 succeeded vs summary was set incorrectly. Can you think of a way this could be done in code – rejecting questions that are likely to be malformed?